Sources/Peer.swift

```diff
@@ -46,6 +46,7 @@ public struct Peer {
             case .notConnected: self = .notConnected
             case .connecting: self = .connecting
             case .connected: self = .connected
+            @unknown default: self = .notConnected
```

Comment:
Maybe remove the case `.notConnected`, as `default` case returns .notConnected as well.

Reply:
`@unknown default:` isn't the same as `default:` so it wouldn't work.

> https://github.com/apple/swift-evolution/blob/master/proposals/0192-non-exhaustive-enums.md